types/zrender/index.d.ts

```diff
@@ -9,7 +9,7 @@ declare namespace zrender {
     type Y = number;
     type X2 = number;
     type Y2 = number;
-    type GlobalCoords = boolean;
+    type Global = boolean;
```

Comment:
This looks correct, but unrelated.  Next time, can you please open a separate PR?

Reply:
this change has been reverted